src/humanloop/client.py

```diff
@@ -78,11 +90,124 @@ def __init__(
             follow_redirects=follow_redirects,
             httpx_client=httpx_client,
         )
+
+        self._tracer_provider = TracerProvider(
+            resource=Resource(
+                attributes={
+                    "instrumentor": "humanloop.sdk",
+                }
+            ),
+        )
+        instrument_provider(provider=self._tracer_provider)
+        self._tracer_provider.add_span_processor(
```

Comment:
Future scope: allowing people to pass in other processors

Reply:
ack - in latest config, they can pass the trace provider so we can attach our own processor